src/main/target/HELIOSPRING/config.c

```diff
@@ -80,12 +80,11 @@ void targetConfiguration(void) {
         pidProfile->itermAcceleratorGain = 5000;
         // should't need to set these since they don't get init in gyro.c with USE_GYRO_IMUF
```

Comment:
If these are really no longer needed we should just remove them entirely instead of leaving comments IMO